.github/workflows/build_images_uv.yaml

```diff
@@ -51,6 +51,7 @@ jobs:
         with:
           platforms: linux/amd64,linux/arm64
           push: true
+          target: "main_image"
           tags: |
```

Comment:
Instead of setting the tags manually, it could be worth checking out https://github.com/docker/metadata-action
It should extract and format most tags like semver more easily.

Reply:
Agree @javiermtorres let's decide how we want to manage the semver, and let's include it in another PR

Reply:
Ok 👍